main/localtime_r.cpp

```diff
@@ -64,6 +65,27 @@ bool ParseSQLdatetime(time_t &time, struct tm &result, const std::string &szSQLd
 	return ParseSQLdatetime(time, result, szSQLdate, ltime.tm_isdst);
 }
 
+/*
+ * Same as ParseSQLdatetime but takes an ISO format (YYYY-MM-DDTHH:mm:ddZ) instead.
+ */
+bool ParseISOdatetime(time_t &time, struct tm &result, const std::string &sISOdate) {
+	if (sISOdate.length() < 20)
+	{
+		return false;
+	}
+
+	std::string sDateWithoutZ = sISOdate.substr(0, 19);
+	std::vector<std::string> splittedDate;
+	boost::split(splittedDate, sDateWithoutZ, boost::is_any_of("T"));
```

Comment:
Is it possible to use the Domoticz StringSplit function from Helper.h ?

Reply:
Yes, it should be fixed now